### PR TITLE
Updated naming conventions around domain names, part 4

### PR DESCRIPTION
**Pull request by orangejenny**
@nickpell

### CODE REVIEW

File: corehq/apps/hqwebapp/views.py [hunk 1]
```diff
@@ -374,14 +374,14 @@ def _login(req, domain_name):
             if template_name is None:
                 template_name = custom_landing_page.get('default', template_name)
     elif domain_name:
-        domain = Domain.get_by_name(domain_name)
+        domain_obj = Domain.get_by_name(domain_name)
         req_params = req.GET if req.method == 'GET' else req.POST
         context.update({
             'domain': domain_name,
-            'hr_name': domain.display_name() if domain else domain_name,
-            'next': req_params.get('next', '/a/%s/' % domain),
```
[the remark this answers is not in the document]
Comment (nickpell) [reply corehq/apps/hqwebapp/views.py]: Looks like it

File: corehq/apps/hqwebapp/views.py [hunk 2]
```diff
@@ -374,14 +374,14 @@ def _login(req, domain_name):
...
-            'allow_domain_requests': domain.allow_domain_requests,
-            'current_page': {'page_name': _('Welcome back to %s!') % domain.display_name()},
+            'hr_name': domain_obj.display_name() if domain_obj else domain_name,
```
Comment (nickpell) [inline corehq/apps/hqwebapp/views.py]: strange that there is a check that ```domain_obj``` is not falsy on this line but not on other lines in this dict where ```domain_obj``` members are accessed.

**Review by nickpell — APPROVED** [review]

**Comment by nickpell** [comment]
Kicked build

File: corehq/apps/hqwebapp/views.py (hunk 2 above)
Comment (orangejenny) [reply corehq/apps/hqwebapp/views.py]: Agreed. My read of this function is that `domain_obj` should always exist, and given that there are other lines that assume it exists, so I just removed this check in 35f5b61495a

**Review by nickpell — APPROVED** [review]

**Pull request merged**